<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefLibraryDlg</class>
 <widget class="QWidget" name="DlgPrefLibraryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>1203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Library Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QGroupBox" name="groupBox_MusicDirectories">
     <property name="title">
      <string>Music Directories</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_music_directories">

      <item row="0" column="0" rowspan="3">
       <widget class="QListView" name="dirList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>

      <item row="0" column="1">
       <widget class="QPushButton" name="PushButtonAddDir">
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>Add a directory where your music is stored. Mixxx will watch this directory and its subdirectories for new tracks.</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>

      <item row="1" column="1">
       <widget class="QPushButton" name="PushButtonRelocateDir">
        <property name="toolTip">
         <string>If an existing music directory is moved, Mixxx doesn't know where to find the audio files in it. Choose Relink to select the music directory in its new location. &lt;br/&gt; This will re-establish the links to the audio files in the Mixxx library.</string>
        </property>
        <property name="text">
         <string extracomment="This will re-establish the links to the audio files in the Mixxx database if you move an music directory to a new location.">Relink</string>
        </property>
       </widget>
      </item>

      <item row="2" column="1">
       <widget class="QPushButton" name="PushButtonRemoveDir">
        <property name="font">
         <font/>
        </property>
        <property name="toolTip">
         <string>If removed, Mixxx will no longer watch this directory and its subdirectories for new tracks.</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>

      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_library_scan">
        <property name="text">
         <string>Rescan directories on start-up</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_AudioFileFormats">
     <property name="title">
      <string>Audio File Formats</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_audio_formats">
      <item row="0" column="0">
       <widget class="QLabel" name="builtInFormats">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_AudioFileTags">
     <property name="title">
      <string>Track Metadata Synchronization</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_metadata">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_SyncTrackMetadata">
        <property name="text">
         <string>Synchronize library track metadata from/to file tags</string>
        </property>
        <property name="toolTip">
         <string>Automatically write modified track metadata from the library into file tags and reimport metadata from updated file tags into the library</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_SeratoMetadataExport">
        <property name="text">
         <string>Synchronize Serato track metadata from/to file tags (experimental)</string>
        </property>
        <property name="toolTip">
         <string>Keeps track color, beat grid, bpm lock, cue points, and loops synchronized with SERATO_MARKERS/MARKERS2 file tags.&lt;br/&gt;&lt;br/&gt;WARNING: Enabling this option also enables the reimport of Serato metadata after files have been modified outside of Mixxx. On reimport existing metadata in Mixxx is replaced with the metadata found in file tags. Custom metadata not included in file tags like loop colors is lost.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_TrackTableView">
     <property name="title">
      <string>Track Table View</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_track_table_view">

      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxEditMetadataSelectedClicked">
        <property name="text">
         <string>Edit metadata after clicking selected track</string>
        </property>
       </widget>
      </item>

      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="doubeClickActionLabel">
        <property name="text">
         <string>Track Double-Click Action:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>

      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="radioButton_dbclick_deck">
        <property name="text">
         <string>Load track to next available deck</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QRadioButton" name="radioButton_dbclick_bottom">
        <property name="text">
         <string>Add track to Auto DJ queue (bottom)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QRadioButton" name="radioButton_dbclick_top">
        <property name="text">
         <string>Add track to Auto DJ queue (top)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QRadioButton" name="radioButton_dbclick_ignore">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>

      <item row="7" column="0">
       <widget class="QLabel" name="label_bpm_precision">
        <property name="text">
         <string>BPM display precision:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QSpinBox" name="spinbox_bpm_precision">
       </widget>
      </item>

     </layout>
    </widget>
   </item><!-- Track Table View -->

   <item>
    <widget class="QGroupBox" name="groupBox_History">
     <property name="title">
      <string>Session History</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_history">

      <item row="1" column="0">
       <widget class="QLabel" name="label_history_track_duplicate_distance">
        <property name="text">
         <string>Track duplicate distance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSpinBox" name="spinbox_history_track_duplicate_distance">
        <property name="toolTip">
         <string>When playing a track again log it to the session history only if more than N other tracks have been played in the meantime</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="label_history_cleanup">
        <property name="toolTip">
         <string>History playlist with less than N tracks will be deleted&lt;br/&gt;&lt;br/&gt;Note: the cleanup will be performed during startup and shutdown of Mixxx.</string>
        </property>
        <property name="text">
         <string>Delete history playlist with less than N tracks</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="spinbox_history_min_tracks_to_keep">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item><!-- History -->

   <item>
    <widget class="QGroupBox" name="groupBox_Miscellaneous">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_misc">

      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_use_relative_path">
        <property name="text">
         <string>Use relative paths for playlist export if possible</string>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="rowHeightLabel">
        <property name="text">
         <string>Library Row Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxRowHeight">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="libraryFontLabel">
        <property name="text">
         <string>Library Font:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="libraryFont">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="libraryFontButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="searchDebouncingTimeoutLabel">
        <property name="text">
         <string>Search-as-you-type timeout:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="searchDebouncingTimeoutSpinBox">
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>

      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxEnableSearchCompletions">
        <property name="text">
         <string>Enable search completions</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxEnableSearchHistoryShortcuts">
        <property name="text">
         <string>Enable search history keyboard shortcuts</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_external_libraries">
     <property name="title">
      <string>External Libraries</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_external_libraries">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_show_rhythmbox">
        <property name="text">
         <string>Show Rhythmbox Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_show_banshee">
        <property name="text">
         <string>Show Banshee Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_show_itunes">
        <property name="text">
         <string>Show iTunes Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_show_traktor">
        <property name="text">
         <string>Show Traktor Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_show_rekordbox">
        <property name="text">
         <string>Show Rekordbox Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_show_serato">
        <property name="text">
         <string>Show Serato Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>All external libraries shown are write protected.</string>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>You will need to restart Mixxx for these settings to take effect.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_cover_art_fetcher">
     <property name="title">
      <string>Preferred Cover Art Fetcher Resolution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_cover_art_fetcher">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Fetch cover art from coverartarchive.com by using Import Metadata From Musicbrainz.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Note: ">1200 px" can fetch up to very large cover arts.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_cover_art_fetcher_highest">
        <property name="text">
         <string>>1200 px (if available)</string>
        </property>
        <property name="checked">
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_cover_art_fetcher_high">
        <property name="text">
         <string>1200 px (if available)</string>
        </property>
        <property name="checked">
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_cover_art_fetcher_medium">
        <property name="text">
         <string>500 px</string>
        </property>
        <property name="checked">
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButton_cover_art_fetcher_lowest">
        <property name="text">
         <string>250 px</string>
        </property>
        <property name="checked">
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_settingsDir">
     <property name="title">
      <string>Settings Directory</string>
     </property>
     <layout class="QVBoxLayout" name="vlayout_settingsDir">
      <item>
       <widget class="QLabel" name="label_settingsDir">
        <property name="text">
         <string>The Mixxx settings directory contains the library database, various configuration files, log files, track analysis data, as well as custom controller mappings.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_settingsManualLink">
        <property name="text">
         <string></string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_settingsWarning">
        <property name="text">
         <string>Edit those files only if you know what you are doing and only while Mixxx is not running.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PushButtonOpenSettingsDir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Open Mixxx Settings Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dirList</tabstop>
  <tabstop>PushButtonAddDir</tabstop>
  <tabstop>PushButtonRelocateDir</tabstop>
  <tabstop>PushButtonRemoveDir</tabstop>
  <tabstop>checkBox_library_scan</tabstop>
  <tabstop>checkBox_SyncTrackMetadata</tabstop>
  <tabstop>checkBox_SeratoMetadataExport</tabstop>
  <tabstop>checkBoxEditMetadataSelectedClicked</tabstop>
  <tabstop>radioButton_dbclick_deck</tabstop>
  <tabstop>radioButton_dbclick_bottom</tabstop>
  <tabstop>radioButton_dbclick_top</tabstop>
  <tabstop>radioButton_dbclick_ignore</tabstop>
  <tabstop>spinbox_bpm_precision</tabstop>
  <tabstop>spinbox_history_track_duplicate_distance</tabstop>
  <tabstop>spinbox_history_min_tracks_to_keep</tabstop>
  <tabstop>checkBox_use_relative_path</tabstop>
  <tabstop>spinBoxRowHeight</tabstop>
  <tabstop>libraryFont</tabstop>
  <tabstop>libraryFontButton</tabstop>
  <tabstop>searchDebouncingTimeoutSpinBox</tabstop>
  <tabstop>checkBoxEnableSearchCompletions</tabstop>
  <tabstop>checkBoxEnableSearchHistoryShortcuts</tabstop>
  <tabstop>checkBox_show_rhythmbox</tabstop>
  <tabstop>checkBox_show_banshee</tabstop>
  <tabstop>checkBox_show_itunes</tabstop>
  <tabstop>checkBox_show_traktor</tabstop>
  <tabstop>checkBox_show_rekordbox</tabstop>
  <tabstop>checkBox_show_serato</tabstop>
  <tabstop>radioButton_cover_art_fetcher_highest</tabstop>
  <tabstop>radioButton_cover_art_fetcher_high</tabstop>
  <tabstop>radioButton_cover_art_fetcher_medium</tabstop>
  <tabstop>radioButton_cover_art_fetcher_lowest</tabstop>
  <tabstop>PushButtonOpenSettingsDir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
